### PR TITLE
Consistently use == over = inside [[

### PR DESCRIPTION
**Pull request by IsaacG**
The test files got roughly 2k `[[ ==` and roughly 450 `[[ x =`.
Make that all-or-none :)

---

Reviewer Resources:

[Track Policies](https://github.com/exercism/bash/blob/master/POLICIES.md)

### CODE REVIEW

File: exercises/allergies/allergies_test.sh [hunk 1]
```diff
@@ -6,49 +6,60 @@
     #[[ $BATS_RUN_SKIPPED == true  ]] || skip
     run bash allergies.sh 0 allergic_to peanuts
     [[ $status -eq 0 ]] 
-    [[ $output = "false" ]]
+    [[ $output == "false" ]]
+    [[ $output == "false" ]]
```
Comment (glennj) [inline exercises/allergies/allergies_test.sh]: Why are these all doubled?

**Comment by glennj** [comment]
Although `=` is still a valid operator, better messaging to students to use `==`.

File: exercises/allergies/allergies_test.sh (hunk 1 above)
Comment (IsaacG) [reply exercises/allergies/allergies_test.sh]: Fixed

**Comment by IsaacG** [comment]
Personally, I prefer `=` over `==` but I went with what we have more of.

**Comment by IsaacG** [comment]
I might have totally messed up my git history at this point. If I can't figure it out, I may need to delete my fork and just start over.

**Comment by IsaacG** [comment]
Many, many, *many* `git rebase` later, I managed to clean up the history and squash all my commits.

**Pull request merged**